src/App.js

```diff
@@ -1,9 +1,20 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { useState } from "react";
+import "./App.css";
+import CountryCard from "./CountryCard";
 
 function App() {
+  const [countriesAll, setCountriesAll] = useState([]);
+
+  fetch("https://restcountries.com/v2/all")
```

Comment:
which React hook do we need to wrap a fetch call in to handle the asynchronous behaviour of fetch?

Reply:
Hi Ellie, in my second PR I wrapped  the fetch in to useEffect hook.